### PR TITLE
WD-11676 - refactor: split exports to enable fast refresh

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- move exports to separate files to enable fast refresh of components
- add index.ts files where required

## Details

https://warthogs.atlassian.net/browse/WD-11676

### CODE REVIEW

File: src/components/Banner/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export { default } from "./Banner";
+export { Label as BannerLabel } from "./types";
```
Comment (vladimir-cucu) [inline src/components/Banner/index.ts:2]: Was wondering if maybe excluding all `index.ts` files from test coverage would make sense in order to not get the Codecov false positive?

**Review by vladimir-cucu — APPROVED** [review]
Looks good! Tested it locally to check for weird behaviors and it works as expected. :rocket:

File: src/components/Banner/index.ts (hunk 1 above)
Comment (huwshimi) [reply src/components/Banner/index.ts:2]: The report actually just revealed a bunch of places I still need to move exports (not sure why the linter didn't catch them). I'll push a change shortly.

Comment (vladimir-cucu) [reply src/components/Banner/index.ts:2]: The changes look good! Thanks.

**Pull request merged**